src/cursor/cur_std.c

```diff
@@ -823,6 +823,7 @@ __wt_cursor_close(WT_CURSOR *cursor)
 
 	session = (WT_SESSION_IMPL *)cursor->session;
 
+	WT_RET(__wt_txn_context_prepare_check(session, false));
```

Comment:
I don't believe WT_CURSOR::close should do a prepare check - we want applications to be able to release resources, and this API call doesn't rely on any prepare semantics.